### PR TITLE
Adjust Simple-Chart block wagtail layout

### PR DESCRIPTION
**Pull request by brandon-rusk**
## Additions

- Made a new layout for the Simple-Chart wagtail block

## Changes

- Slight text change in one of the Simple-Chart fields

## How to test this PR

1. Pull PR
2. `yarn build` -- `./runserver.sh`
3. Go to http://localhost:8000/admin/pages/4481/edit/
4. Add Simple-Chart block and notice new layout

## Screenshots

image

image

Resolves https://github.cfpb.gov/CFPB/super-regular/issues/383

### CODE REVIEW

File: cfgov/unprocessed/css/on-demand/simple-chart-admin.less [hunk 1]
```diff
@@ -0,0 +1,25 @@
+  @media (min-width: 970px) {
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/unprocessed/css/on-demand/simple-chart-admin.less:1]: Yes I think `[wagtailadmin/css](https://github.com/cfpb/consumerfinance.gov/tree/main/cfgov/templates/wagtailadmin/css)` is the only spot.

File: cfgov/unprocessed/css/on-demand/simple-chart-admin.less [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  .struct-block.simple-chart-block {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: -1.5em;
+  }
+
+  .struct-block.simple-chart-block > .field {
```
Comment (contolini) [inline cfgov/unprocessed/css/on-demand/simple-chart-admin.less:8]: Is it necessary to have `.struct-block` in front of `.simple-chart-block`? Is just `.simple-chart-block` not enough specificity?

File: cfgov/unprocessed/css/on-demand/simple-chart-admin.less (hunk 1 above)
[the remark this answers is not in the document]
Comment (brandon-rusk) [reply cfgov/unprocessed/css/on-demand/simple-chart-admin.less]: Thanks all, I have made the appropriate changes!

**Comment by ielerol** [comment]
I have some thoughts to share, some specific to the two-column layout approach, some are more general summaries of feedback we've gotten about the Wagtail admin that I'm including here because it seems like a good opportunity to make those changes.

As far as layout goes, I think the most useful thing to do is keep fields that are likely to be edited together as close together as possible. To a certain degree this is already true, but there are a few places where I think we can rearrange things to be slightly more logical. For example, my experience so far has been that axis labels are likely to change independently of data sources and structure, as a result of content clearance, so: 

-Switch the position of "Transform" and "Y-axis label", so that the two axis label fields are next to each other.

The other thing that would help make editing simple is to match the order of the fields in the admin to the order they appear on the website, as much as possible. I don't think it makes sense to move the figure number field from where it is right now, since it is optional and I find that scanning for the chart title is the most important thing to look for when I'm editing pages with multiple charts, and it's about as prominent as it can be right now. But for the fields that appear below charts, the order and labeling should match what's live, and also how the information is arranged in content tables, to make it as simple as possible for content editors to copy and paste accurately.

- "Credits" field should be renamed "Source credits" (to minimize possible confusion with the "data source" field)
- Fields for information that appears below the chart should go in the order Source credits, Date published, Download text, Download file, Notes

This does require moving the notes fields which unbalances the columns a bit, but I think the tradeoff of page length is well worth it in terms of following the structure of how the content actually appears. It might also be nice to make the notes field full-width, for ease of reading longer note text.

The other things are just minor improvements to the helper text for some fields. 
- Change the helper text for "Data series" to: `For charts pulling from a separate source file, include a list of the column headers (from a CSV file) or keys (from a JSON file) that match the data that needs to appear in the chart, in the format ["HEADER/KEY1", "HEADER/KEY2", "HEADER/KEY3"]. If the chart labels need to be different than the data source headers, include the correct labels with the format [{"key": "HEADER/KEY1", "label": "NEWLABEL1"}, {"key": "HEADER/KEY2", "label": "NEWLABEL2"}]`
- Change the helper text for "Filters" to: `If the chart needs the option for users to filter the data shown, for example by date or geographic region, provide the JSON objects to filter on, in the format {key: "KEY", "label": "LABEL"}`

File: cfgov/templates/wagtailadmin/css/simple-chart-admin.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  .struct-block.simple-chart-block > .field[data-contentpath="style_overrides"] {
+    min-width: calc(100% - 1.5em);
+  }
+
+  .struct-block.simple-chart-block > .field[data-contentpath="download_file"] {
+    margin-left: 0;
+  }
+ }
```
Comment (contolini) [inline cfgov/templates/wagtailadmin/css/simple-chart-admin.css:33]: The formatting is off in this file and now that we have stylelint (!!) we can autoformat it. In package.json try changing [L67](https://github.com/cfpb/consumerfinance.gov/blob/0b949a2b261de160b51b57b820c39a7bf65e1810/package.json#L67) to:

```js
"lint": "eslint --ignore-pattern node_modules './{cfgov/unprocessed,config,esbuild,scripts,test}/**/*.js' --fix && stylelint './cfgov/{unprocessed,templates/wagtailadmin/css}/**/*.{css,less}' --fix",
```

and then run `yarn lint` to fix all our custom wagtail admin CSS files along with our normal Less files.

**Pull request merged**